x-pack/legacy/plugins/apm/server/lib/service_map/get_service_map.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PromiseReturnType } from '../../../typings/common';
+import {
+  Setup,
+  SetupTimeRange,
+  SetupUIFilters
+} from '../helpers/setup_request';
+import { getServiceMapFromTraceIds } from './get_service_map_from_trace_ids';
+import { getTraceSampleIds } from './get_trace_sample_ids';
+import { getServicesProjection } from '../../../common/projections/services';
+import { mergeProjection } from '../../../common/projections/util/merge_projection';
+import {
+  SERVICE_AGENT_NAME,
+  SERVICE_NAME
+} from '../../../common/elasticsearch_fieldnames';
+
+export interface IEnvOptions {
+  setup: Setup & SetupTimeRange & SetupUIFilters;
```

Comment:
this should not include `SetupUIFilters`

Reply:
it's still needed for the `getServicesProjection` call